lib/cloudtasker/engine.rb

```diff
@@ -11,6 +11,14 @@ class Engine < ::Rails::Engine
       end
     end
 
+    config.before_configuration do
+      require 'active_job/queue_adapters/cloudtasker_adapter'
+    end
+
+    config.after_initialize do
```

Comment:
So assuming `worker` has been moved to `cloudtasker_adapter` we could replace these two blocks by:

```ruby
initializer "cloudtasker.active_job_integration" do
    require 'active_job/queue_adapters/cloudtasker_adapter' if defined?(::ActiveJob::Railtie)
end
```

Reply:
I would still have problems loading the code.

Reply:
But I'll still be adding the conditional loading of the adapter.

Reply:
(Though not using `if` as a modifier...)